internal/binutils/binutils.go

```diff
@@ -120,6 +121,29 @@ func (bu *Binutils) SetTools(config string) {
 	bu.update(func(r *binrep) { initTools(r, config) })
 }
 
+func ensureObjdumpVersion(objdump string, objdumpFound bool) (string, bool) {
```

Comment:
Prefer top-down function order in Go - place this function after its use, i.e. after initTools.

Reply:
done.